Kernel/Arch/aarch64/vector_table.S

```diff
@@ -60,10 +60,12 @@
     str x0, [sp, #ELR_EL1_SLOT]
     mrs x0, tpidr_el0
     str x0, [sp, #TPIDR_EL0_SLOT]
+    mrs x0, sp_el0
+    str x0, [sp, #SP_EL0_SLOT]
```

Comment:
do we need to store the userspace stack when switching to the kernel calls?

(i guess iʼm assuming that we donʼt do any process/thread switching and if it is done, then it is made explicitly in a designated place? 🤔)

Reply:
Yes, we definitely need to store the current SP_EL0 stack pointer. I'm not really sure I understand you question, but we can discuss about this on Discord :)